tests/rgw/sanity_rgw_multisite.py

```diff
@@ -101,6 +104,58 @@ def run(**kw):
     return 0
 
 
+def sync_status(verify_io_on_site_node, retry=5, delay=30):
```

Comment:
can you remove this from here put in utlis, and also possibly rename the function there. we can make use of this in other tests file in the future.